R/parse_url.R

```diff
@@ -1,18 +1,23 @@
-#' Creates Shinylive url for the app code.
+#' Creates Shinylive url based on app code.
 #'
 #' @importFrom jsonlite unbox toJSON
 #' @importFrom lzstring compressToEncodedURIComponent
 #'
 #' @param code (`character(1)`) A string with app code.
+#' @param mode (`character(1)`) A string with mode. One of "app" or "editor".
+#' @param header (`logical(1)`) A logical value indicating whether to include header.
+#' Ignored if `mode` is "editor".
 #' @return (`character(1)`) Shinylive app url.
 #'
 #' @export
 #'
 #' @examples
 #' code <- "this is your app code as a string"
 #' create_shinylive_url(code)
```

Comment:
```suggestion
#' create_shinylive_url(code)
#' create_shinylive_url(code, mode = "editor")
#' create_shinylive_url(code, header = FALSE)
```

Should we consider adding more examples using the new parameters?

Reply:
yes - added!